package/debian/opt/thingsplex/easee/defaults/config.json

```diff
@@ -23,5 +23,9 @@
     "keepAliveInterval2": "30s",
     "timeoutInterval2": "1m",
     "invokeTimeout": "10s"
+  },
+  "backoff": {
```

Comment:
as you're extending the config here, please register the routes to be able to manipulate those values via MQTT.